{ 'pack_motion': { 'head': [],
                   'left_arm': [ { 'duration': 1.975,
                                   'name': u'ready_state',
                                   'positions': [ -1.5135,
                                                  0.7532,
                                                  -0.2755,
                                                  0.0212,
                                                  0.0208,
                                                  -0.2499,
                                                  0.8168],
                                   'starttime': 0.0},
                                 { 'duration': 1.975,
                                   'name': u'pack_1',
                                   'positions': [ -2.8986,
                                                  0.7301,
                                                  -0.2675,
                                                  0.0351,
                                                  1.0944,
                                                  -0.0002,
                                                  0.82],
                                   'starttime': 1.99},
                                 { 'duration': 1.975,
                                   'name': u'pack_final',
                                   'positions': [ -3.1165,
                                                  -0.5124,
                                                  0.0,
                                                  0.0,
                                                  1.885,
                                                  -0.0,
                                                  -0.0],
                                   'starttime': 7.011}],
                   'left_leg': [ { 'duration': 2.04,
                                   'name': u'ready_state',
                                   'positions': [ 0.0113,
                                                  -0.0746,
                                                  -0.7899,
                                                  1.1893,
                                                  0.6074,
                                                  -0.0704],
                                   'starttime': 0.0},
                                 { 'duration': 2.95,
                                   'name': u'pack_final',
                                   'positions': [ -0.002,
                                                  0.0012,
                                                  -1.5709,
                                                  3.1416,
                                                  1.8204,
                                                  0.0001],
                                   'starttime': 4.015}],
                   'right_arm': [ { 'duration': 1.975,
                                    'name': u'ready_state',
                                    'positions': [ 1.5135,
                                                   -0.7532,
                                                   0.2755,
                                                   -0.0212,
                                                   -0.0208,
                                                   0.2499,
                                                   -0.8168],
                                    'starttime': 0.0},
                                  { 'duration': 1.975,
                                    'name': u'pack_1',
                                    'positions': [ 2.8986,
                                                   -0.7301,
                                                   0.2675,
                                                   -0.0351,
                                                   -1.0944,
                                                   0.0002,
                                                   -0.82],
                                    'starttime': 1.99},
                                  { 'duration': 1.975,
                                    'name': u'pack_final',
                                    'positions': [ 3.1165,
                                                   0.5124,
                                                   -0.0,
                                                   -0.0,
                                                   -1.885,
                                                   0.0,
                                                   0.0],
                                    'starttime': 6.995}],
                   'right_leg': [ { 'duration': 2.04,
                                    'name': u'ready_state',
                                    'positions': [ -0.0113,
                                                   0.0746,
                                                   0.7899,
                                                   -1.1893,
                                                   -0.6074,
                                                   0.0704],
                                    'starttime': 0.0},
                                  { 'duration': 2.95,
                                    'name': u'pack_final',
                                    'positions': [ 0.002,
                                                   -0.0012,
                                                   1.5709,
                                                   -3.1416,
                                                   -1.8204,
                                                   -0.0001],
                                    'starttime': 4.015}],
                   'torso': [ { 'duration': 1.975,
                                'name': u'ready_state',
                                'positions': [0.0001, -0.0002],
                                'starttime': 0.0},
                              { 'duration': 3.015,
                                'name': u'pack_final',
                                'positions': [-0.0, 1.5728],
                                'starttime': 9.017}]}}
